mods/ts/rules/defaults.yaml

```diff
@@ -668,6 +668,31 @@
 		Terrain: Tree
 	Tooltip:
 		Name: Tree
+	Health:
+		HP: 500
+	Armor:
+		Type: Wood
+	Targetable:
+		TargetTypes: Trees
+	WithDamageOverlay@SmallBurn:
+		DamageType: Incendiary
+		Image: burn-s
+		MinimumDamageState: Light
+		MaximumDamageState: Dead
+		Palette: effect
+	WithDamageOverlay@MediumBurn:
+		DamageType: Incendiary
+		Image: burn-m
+		MinimumDamageState: Medium
+		MaximumDamageState: Dead
```

Comment:
In TS it was possible to 'chain' animations. The fire anims didn't play at the same time.

https://github.com/LipkeGu/TSIniFiles/blob/master/rules.ini#L354 is just a randomizer for the initial flame anim.